### ISSUE TITLE
 Paste before selecting a copy file and down

### ISSUE THREAD

**Issue by tenderness330** [skanehira/ff#12]
OS: WSL ubuntu 18.04
go  version: go1.13.4 linux/amd64

If you enter the p key before selecting a copy file, ff goes down with an exception.
If you put it after yunk, it will work without problems.

error messages.
```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
        panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x7660bd]

goroutine 1 [running]:
github.com/rivo/tview.(*Application).Run.func1(0xc000265000)
        /home/user/go/pkg/mod/github.com/rivo/tview@v0.0.0-20190324182152-8a9e26fab0ff/application.go:149 +0x82
panic(0x7a5160, 0xb7efc0)
        /usr/local/go/src/runtime/panic.go:679 +0x1b2
github.com/skanehira/ff/gui.(*Gui).EntryManagerKeybinding.func2(0xc0003d0540, 0xc000290001)
        /home/user/ff/gui/keybindings.go:159 +0x4ad
github.com/rivo/tview.(*Box).WrapInputHandler.func1(0xc0003d0540, 0xc000290090)
        /home/user/go/pkg/mod/github.com/rivo/tview@v0.0.0-20190324182152-8a9e26fab0ff/box.go:149 +0x75
github.com/rivo/tview.(*Application).Run(0xc000265000, 0x0, 0x0)
        /home/user/go/pkg/mod/github.com/rivo/tview@v0.0.0-20190324182152-8a9e26fab0ff/application.go:234 +0x415
github.com/skanehira/ff/gui.(*Gui).Run(0xc0001e0700, 0xc0001e0700, 0x7aad00)
        /home/user/ff/gui/gui.go:207 +0x7da
main.run(0xc000098120)
        /home/user/ff/main.go:33 +0xb7
main.main()
        /home/user/ff/main.go:66 +0x79
```

**Comment by skanehira** [comment]
@tenderness330 Thank you 👍 
I fixed this problem in f16050e9a0de56340fa164eb74c53070c22514d1

**Comment by tenderness330** [comment]
I tried update and just first push key p. ff didn't downed with an exception.
Thank you for fix.
